src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/VariantEval.java

```diff
@@ -488,6 +492,7 @@ public void onComplete() {
 
     @Override
     public void apply(VariantContext variant, ReadsContext readsContext, ReferenceContext referenceContext, FeatureContext featureContext) {
+        referencePositionsCoveredByVariants += variant.getLengthOnReference();
```

Comment:
Won't you double-count territory for overlapping variants? Does this matter?

Reply:
As discussed in person, it might be preferable, instead of returning a nonsense value here, to add a `requiresReference()` (or similar) method to the `VariantEvaluator` class, interrogate the selected evaluations in `onTraversalStart()` in `VariantEval`, and throw an error if any of the selected evaluations require a reference.